commons/monitor/questionnaire/questionnaire_application_interaction_event.avsc

```diff
@@ -7,13 +7,14 @@
     { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
     { "name": "eventType", "type": {
       "name": "InteractionEventType",
-      "doc": "NOTIFICATION_OPEN = When a user presses the notification to open the app, APP_OPEN = When a user opens the app directly, QUESTIONNAIRE_STARTED = When a user starts a questionnaire, QUESTIONNAIRE_FINISHED = When a user completes a questionnaire, QUESTIONNAIRE_CANCELLED = When a user closes a questionnaire without completing it.", 
+      "doc": "NOTIFICATION_OPEN = When a user presses the notification to open the app, APP_OPEN = When a user opens the app directly, QUESTIONNAIRE_STARTED = When a user starts a questionnaire, QUESTIONNAIRE_FINISHED = When a user completes a questionnaire, QUESTIONNAIRE_CANCELLED = When a user closes a questionnaire without completing it., QUESTIONNAIRE_TRIGGERED = When a data message triggers a new questionnaire.", 
       "type": "enum",
-      "symbols": ["NOTIFICATION_OPEN", "APP_OPEN", "QUESTIONNAIRE_STARTED", "QUESTIONNAIRE_FINISHED", "QUESTIONNAIRE_CANCELLED", "OTHER", "UNKNOWN", "RECORDING_STARTED", "RECORDING_STOPPED"]
+      "symbols": ["NOTIFICATION_OPEN", "APP_OPEN", "QUESTIONNAIRE_STARTED", "QUESTIONNAIRE_FINISHED", "QUESTIONNAIRE_CANCELLED", "OTHER", "UNKNOWN", "RECORDING_STARTED", "RECORDING_STOPPED", "QUESTIONNAIRE_TRIGGERED"]
       },
       "doc": "Questionnaire app activity usage event type.",
       "default": "UNKNOWN"
     },
-    { "name": "questionnaireName", "type": ["null", "string"], "doc": "Name of the questionnaire.", "default": null }
+    { "name": "questionnaireName", "type": ["null", "string"], "doc": "Name of the questionnaire.", "default": null },
+    { "name": "metadata", "type": ["null", "string"], "doc": "Event metadata.", "default": null }
```

Comment:
I would also add to the doc that, for the `QUESTIONNAIRE_TRIGGERED ` event, this will be used to propagate any metadata received in the data message payload.

Reply:
Indeed. Also in doc, what format this metadata will have.

Reply:
@blootsvoets it can be a Map with any K,V pairs based on the output of the inference. This is just so we can know which questionnaire was triggered by which model and params. But i agree @mpgxvii, would be good to include this in the docs as well.

Reply:
If this list of params is more or less fixed, you could opt to just make this an actual K,V map (`{"type": "map","values":["null","string"]}`. The output CSV files will then put data with and without these parameters into separate files. Alternatively, please specify the syntax of the K,V map (e.g., JSON).

Reply:
It will not be fixed and could vary with the model used for analysis. I agree we can perhaps make this a map.

Reply:
Okay sure I'll make the changes